arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+{
+    "swagger": "2.0",
+    "host": "management.azure.com",
+    "info": {
+        "version": "2016-01-29",
+        "title": "Power BI Embedded Management Client",
+        "description": "Azure Resource Provider for Power BI Embedded"
+    },
+    "schemes": [
+        "https",
+        "http"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections/{workspaceCollectionName}": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "WorkspaceCollections_getByName",
+                "description": "The **Get Power BI Workspace Collection** operation retrieves an existing Power BI Workspace Collection.",
```

Comment:
There are some operations that work on groups of workspaceCollection items such as listBySubscription so I thought was better to separate those from the operations which work on a single workspaceCollection such as delete, but I can see it either way.  I will change to make them all use plural form as `WorkspaceCollections_`
Will also remove all the ** in descriptions. I don't know why this was there either.  I assume it side affect of someone copying from Markdown template.  This swagger file was manually created.